src/platform_specific/affinity.win.cc

```diff
@@ -11,11 +11,11 @@ namespace detail {
 	uint32_t affinity_cores_available() {
 		using native_cpu_set = DWORD_PTR;
 
-		native_cpu_set affinity_base_mask;
+		native_cpu_set available_cores;
 		[[maybe_unused]] native_cpu_set sys_affinity_mask;
-		const auto base_mask_error = GetProcessAffinityMask(GetCurrentProcess(), &affinity_base_mask, &sys_affinity_mask);
-		assert(base_mask_error == 0 && "Error retrieving base affinity mask.");
-		return util::popcount(affinity_base_mask);
+		const auto affinity_error = GetProcessAffinityMask(GetCurrentProcess(), &available_cores, &sys_affinity_mask);
+		assert(affinity_error == 0 && "Error retrieving base affinity mask.");
```

Comment:
This is a bug. `GetProcessAffinityMask` returns `TRUE` (yes, all caps, because it's of type `BOOL`) on success, so this `assert` will fail if the function call was successful.

Reply:
(Also the message is still referring to a "base" affinity mask)